kinetic/distribution.yaml

```diff
@@ -2248,6 +2248,21 @@ repositories:
       url: https://github.com/UbiquityRobotics/dnn_detect.git
       version: kinetic-devel
     status: developed
+  dockeros:
+    doc:
+      type: git
+      url: https://github.com/ct2034/dockeROS.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/ct2034/dockeros-git.git
```

Comment:
This url is incorrect. I found a release repo at dockeros-release.git but that repository does not appear to have had any pushes from running bloom-release so even with the updated url it would still fail to pass CI.

Reply:
Sorry, this is now only about the added python library